docs/deployment.md

```diff
@@ -95,16 +95,15 @@ With that in hand, you can follow the instruction in the [`frontend README`](app
 In the final call to `yarn run dev`, adapt the URLs in the environment variables to match the output of the service tunnel commands.
 
 :warning: the link provided by the client to download samples won't work when Chainsail is deployed via Minikube, the reason being that the host machine does not see the Minikube-internal DNS server by default. To download sampling results, use the following command:
-```console
-kubectl exec -it scheduler-worker-<tab complete here> -- \
-  curl --output - "<URL from download button>" > results.zip
+```bash
+kubectl exec minio-0 -- curl --output - '<URL from download button>' >results.zip
```

Comment:
Nice, good to know that `minio-0` also works! :slightly_smiling_face:

Reply:
Yes. I looked for one that didn't rely on tab-completion. I wrote a little script to test this while I was working on the problem and having a fixed pod was easier 😄.